integration/kubernetes/k8s-block-volume.bats

```diff
@@ -48,8 +48,10 @@ setup() {
 	# Verify persistent volume claim is bound
 	kubectl get pvc | grep "Bound"
 
+	# Add the e2fsprogs package to get mkfs.ext4 installed
+	kubectl exec "$pod_name" -- sh -c "apk add --no-cache e2fsprogs"
 	# make fs, mount device and write on it
-	kubectl exec "$pod_name" -- sh -c "mkfs -t ext4 $ctr_dev_path"
+	kubectl exec "$pod_name" -- sh -c "mkfs.ext4 $ctr_dev_path"
```

Comment:
I don't see an explanation for this part of the change?

Reply:
merely that Alpine apk install does not install the `mkfs` generic command, so we invoke the specific `mkfs.ext4` directly - does it need calling out in the code etc.?

Reply:
Well I personally didn't know alpine didn't provide the wrapper command so I'd argue it isn't obvious. I speculated this might be the reason, but if you document it, it will be clear to all ;)

Reply:
I've added the following text to the commit message, as I don't think having a comment in the code (describing a historical change) will add any extra clarity there:

```
The original test invoked `mkfs` with an ext4 option, but
the generic `mkfs` is not available in the Alpine package,
so invoke `mkfs.ext4` directly instead.
```